### ISSUE TITLE
Getting an error when trying to connect to Hypixel

### ISSUE THREAD

**Issue by CodingCreeperDude96** [PrismarineJS/mineflayer#601]
Hello, I am using your NPM package right now and while I try joining Hypixel it throws this error from the events.js file:

Deserialization error for play.toClient : Read error for name : 960204 is not in the mappings value

I can't edit the file so this stops crushing my program, it's just a server error I want to ignore right now.
Is it possible to ignore this or catch it? because it seems like event listeners don't work.

**Comment by roblabla** [comment]
This is a protocol error. It is probably not possible to recover from it, as it means we have failed to properly understand a server packet. We're aware that node-minecraft-protocol (and thus mineflayer) choke on servers using BungeeCord, such as Hypixel. I'll try to debug the issue when I have some free time.

**Comment by CodingCreeperDude96** [comment]
Oh cool ok, I can't do anything atm?

**Comment by roblabla** [comment]
Well, if you got the technical know-how to debug the issue, I could help you and guide you through the code. Otherwise, yeah, I'm fairly busy right now but I'll try to figure out what's going on when I can.

**Comment by Wqrld** [comment]
Bump, bungee support would be gr8

**Comment by SuperOP535** [comment]
bungee is supported tho

**Comment by rom1504** [comment]
Not really. I never tested on bungee and people tend to report issues when
they try it.

**Comment by SuperOP535** [comment]
I tested and it worked just fine

**Comment by raphydaphy** [comment]
I'm having trouble connecting to minesuperior for presumably the same reason

**Comment by Endum** [comment]
> I'm having trouble connecting to minesuperior for presumably the same reason
> 
> Error: PartialReadError: Deserialization error for status.toClient.server_info.response.$count : Read error for params.server_info.response.$count : undefined

Same here, 1 year is gone, could you please fix that?
